### PR TITLE
Change Template_Params to Params

### LINKED ISSUE #282: Changed `Template_Params` into `Params`

### PR DESCRIPTION
**Pull request by nemanja-boric-sociomantic**
ddoc uses Params both for template and runtime function parameters, but
the existing documentation used `Template_Params` for the template
parameters' documentation. They are now unified under `Params`.

Fixes #282

### CODE REVIEW

File: src/ocean/core/Traits.d [hunk 1]
```diff
@@ -149,7 +149,7 @@ public bool isIdentifier ( cstring input )
 
     If T is enum, aliases to its base type. Otherwise aliases to T.
 
-    Template_Params
+    Params
```
Comment (gavin-norman-sociomantic) [inline src/ocean/core/Traits.d]: Missing :

**Review by gavin-norman-sociomantic — CHANGES_REQUESTED** [review]
Just one mini point.

**Review by mihails-strasuns-sociomantic — APPROVED** [review]

File: src/ocean/core/Traits.d (hunk 1 above)
Comment (nemanja-boric-sociomantic) [reply src/ocean/core/Traits.d]: Good catch.

**Comment by nemanja-boric-sociomantic** [comment]
Updated.

**Review by gavin-norman-sociomantic — APPROVED** [review]

**Review by leandro-lucarella-sociomantic — APPROVED** [review]

**Comment by leandro-lucarella-sociomantic** [comment]
It would be nice to generate the docs somewhere to verify this works :)

**Pull request merged**